quickstarts/delphix/config.yml

```diff
@@ -0,0 +1,39 @@
+# Name of the quickstart (required)
+name: Delphix
+
+# Displayed in the UI (required)
+title: Delphix Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  The template quickstart allows you to get visibility into all your Delphix Virtualization platforms. Use this quickstart together with the mock up integrations.
```

Comment:
```suggestion
  This Delphix quickstart allows you to get visibility into all your Delphix Virtualization platforms. Use this quickstart together with the mock up integrations.
```

Reply:
Just a small suggested wording change but I also have another question: Do you think it is clear what `mock up integrations` refers to? I would consider rephrasing this, I assume the goal is for the user to follow the GitHub readme file?

Reply:
Hello,

Thanks for the comment! yes, i can update this comment from --> "mock up integrations" to "following integrations: <https://github.com/delphix/dct-newrelic-integration>

Reply:
I made a suggestion in the commit but I'd like to see this link being referenced in markdown format if you can change it that would be great.